service/raspbian-install-device-agent.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root"
```

Comment:
The flowfuse install script uses sudo for any actions that need root. On raspbian this works without needing a password, but the script will ask for password if needed

https://github.com/flowforge/installer/blob/a7909d1074106fa3f12abf3c8a4241eb6dcda44b/install.sh#L6

Reply:
Now all the commands are being run with `sudo` we shouldn't need  this warning, but we should swap the test to the negative and show a message saying it may ask for a password to run sudo only if not being run as root.